src/core/game-camera.js

```diff
@@ -33,18 +36,59 @@ export class GameCamera {
   get x() {
     return this.#x
   }
-  set x(val) {
-    gameMouse.x += val - this.#x
-    gameDrawing.moveCamera(val, this.#y)
-    this.#x = val
-  }
```

Comment:
Keep all camera movement logic in a single place, `moveTo` instead of `x` and `y` plus `moveTo`.